styles.css

```diff
@@ -106,10 +115,410 @@ svg {
+  font-weight: 400;
+  font-size: 1em;
+  line-height: 24px;
+  color: #fff;
+}
+
+.project {
+  width: 100%;
+  height: 3em;
+  background-color: var(--cardbutton);
+  border: none;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+}
+
+.project:hover {
+  background-color: #efa628fc;
+  border: 2px solid orangered;
+  cursor: pointer;
+}
+
+.project:active {
+  background-color: var(--cardbutton);
+}
+
+.project:disabled {
+  background-color: #8993a4;
+}
+
+.project a {
+  text-decoration: none;
+  color: #fff;
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+}
+
+/* styles for the about me section */
+.about {
+  display: flex;
+  justify-content: center;
+  background-color: #1c1a19;
+}
+
+.about_wrapper {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  justify-content: center;
+  margin-top: 6.3em;
+  width: 90%;
+  height: 100%;
+  background-repeat: no-repeat;
+  background-position: center right;
+}
+
+.about_wrapper .aboutme {
+  color: var(--cardbutton);
+  font-family: 'Crete Round', serif;
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 88px;
+}
+
+.about_wrapper .aboutparagraph {
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 1em;
+  line-height: 24px;
+  color: #f4f5f7;
+}
+
+.about_wrapper button {
+  background-color: var(--cardbutton);
+  width: 9.75em;
+  height: 3em;
+  border: none;
+  margin-top: 1.375em;
+  padding: 0.75em;
+}
+
+.about_wrapper button:hover {
+  background-color: #efa628fc;
+  border: 2px solid orangered;
+  cursor: pointer;
+}
+
+.about_wrapper button:active {
+  background-color: var(--cardbutton);
+}
+
+.about_wrapper button:disabled {
+  background-color: #8993a4;
+}
+
+.about_wrapper button a {
+  color: #fff;
+  text-decoration: none;
+}
+
+.aboutimage {
+  display: flex;
+  justify-content: space-between;
+}
+
+.container {
+  /* background-color: blue; */
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-bottom: 40px;
+}
+
+.container .title {
+  display: flex;
+}
+
+.title h1 {
+  color: #fff;
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: 2em;
+}
+
+.title .imagelable {
+  width: 18px;
+  height: 18px;
+  margin-top: 7px;
+  margin-right: 10px;
+}
+
+.below ul {
+  text-align: center;
+}
+
+.below ul li {
+  list-style: none;
+  line-height: 42.67px;
+  color: #fff;
+}
+
+/*
+.bottom {
+  padding-top: 40px;
+  background-color: white;
+  width: 100%;
+}
+
+.bottom .bottomwords {
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  display: flex;
+  width: 90%;
+  height: 216px;
+  font-size: 2em;
+  margin-left: auto;
+  margin-right: auto;
+  line-height: 44px;
+  color: #172b4d;
+  text-align: center;
+}
+
+.form {
+  width: 90%;
+  margin-left: auto;
+  margin-right: auto;
+  padding-top: 3.75em;
+}
+
+.form input {
+  width: 100%;
+  border-top: none;
+  border-left: none;
+  border-right: none;
+  border-bottom: 2px solid  #dbd8d7;
+}
+
+.form textarea {
```

Comment:
- [ ]  Kindly remove all commented pieces of code from your documents. This will make your work clean and more professional.